### ISSUE TITLE
Can't figure out why checksum is invalid

### ISSUE THREAD

**Issue by Crusa1der** [sanni/cartreader#1038]
### OSCR firmware version

2023 Version 13.1

### OSCR hardware version

Select hardware version

### System used

SNES/Super Famicom

### Describe the issue.

Hey, I'm new to emulation in general and have been trying to get Super Metroid to work but keep getting the message of an invalid checksum. 

Title: Superman Metroid
Revision: 0
Type: LoROM FastROM
IC's: ROM SAVE
Rom size: 3MB (96 banks)
Save Size: 8 KB
Checksum: F8DF
Press button...

Read ROM

Saving to SNES/ROM/Super Metroid/5/...
[**************]
Checksum... 76F0 != F8DF
Invalid Checksum
CRC32...493C9489 -> Not found
Press button...

It will then save a rom labeled Super Metroid.sfc as well as a text file of the same name and I've tried running it on both higan (which loads the game but doesn't output video or sound [shows fps and game title]), and Snes9x (which instantly tells me it's an invalid checksum). Any help would be appreciated. Also, I have not tried with any other cartridge, if that helps.

**Comment by PsychoFox11** [comment]
Do you get a consistent CRC32 on multiple dumps? I haven't checked this one and doubt a first party title would have an invalid checksum, but some games do - the console doesn't care. But the fact that it isn't running right in emulators tells me the dump is bad.
Make sure it is clean as heck with some 99% alcohol and q-tips, till they are white. The OSCR is far pickier than real consoles about dirt.
If you aren't getting a consistent CRC32 on each dump something is definitely not right with the connection (though in some cases even if it is consistent there could be a connection issue, but when it isn't, there definitely is).

**Comment by Crusa1der** [comment]
I cleaned the cart until there was no black or discoloration on the q-tip and tried again, got the same exact invalid checksum 
CRC32...493C9489 -> Not found
Also, sorry for the late reply, didnt have any rubbing alcohol.

**Comment by PsychoFox11** [comment]
Are you sure it is a legit cart? Got a pic of the PCB? Even a bootleg with wrong CRC32 should load in an emulator though. Maybe with a checksum warning, but it would work.

**Comment by Crusa1der** [comment]
![Uploading 20241024_170803.jpg…]()

**Comment by Crusa1der** [comment]
> I just noticed the "Superman Metroid" - something isn't right there

No, it does say super metroid. My autocorrect was on. 
![20241023_195034](https://github.com/user-attachments/assets/4e48ca50-bd36-4b7f-9cde-7ecc5dd8e21d)

**Comment by Ancyker** [comment]
Are you on the Discord?

**Comment by Crusa1der** [comment]
> Are you on the Discord?

No, I don't really use those kinds of apps. Let me try to resend the photo
![20241024_170803](https://github.com/user-attachments/assets/4e635862-18fe-411c-ba6b-41777bd110d6)

**Comment by PsychoFox11** [comment]
Other SNES games dump ok? Especially any with similar settings?

**Comment by Crusa1der** [comment]
> Other SNES games dump ok? Especially any with similar settings?

I have not tried other snes or sfc games because I dont have others, but I can get another sometime next week or so.

**Comment by PsychoFox11** [comment]
Yeah you really need more than one to confirm if something with the dumper may be wrong. Ideally a few to test. Could be bad cart, bad OSCR, bad anything at this point.

**Comment by Barawer** [comment]
If you are using a computer port or something similar to power the OSCR try using a good wall usb charger. I've had issues like this due to not enough power.

**Comment by PsychoFox11** [comment]
Yeah I agree checking things like that, plus running the self check in the menu if you haven't, would be a good start - if other's don't dump well. Do carts for any console dump properly?

**Comment by Crusa1der** [comment]
I just got another game to try, sorry for the wait.
Still invalid checksum, but it seems diffrent
![20241029_172112](https://github.com/user-attachments/assets/e052a293-1c16-48b0-8c0f-3cfdbaee9709)

**Comment by Crusa1der** [comment]
![20241029_171943](https://github.com/user-attachments/assets/de74114f-bb2a-484c-b6db-0eb44cec852b)

**Comment by Crusa1der** [comment]
I also cleaned the cart before trying so that can't be it.

**Comment by PsychoFox11** [comment]
I'd guess the problem is in your OSCR somewhere, rather than the cart. Does a cart for a different system dump? Unless the problem is on one or more pins that only SNES uses, it would typically affect other consoles.
